.travis.yml

```diff
@@ -41,6 +41,8 @@ env:
     - MYSQL_IMAGE=mysql:5.6 MONGO_IMAGE=mongo:3.6 POSTGRES_IMAGE=postgres:10 PMM_SERVER_IMAGE=percona/pmm-server:2.0.0
     - MYSQL_IMAGE=mysql:5.7 MONGO_IMAGE=mongo:4.0 POSTGRES_IMAGE=postgres:11 PMM_SERVER_IMAGE=percona/pmm-server:2.0.1
     - MYSQL_IMAGE=mysql:8.0 MONGO_IMAGE=mongo:4.2 POSTGRES_IMAGE=postgres:12 PMM_SERVER_IMAGE=percona/pmm-server:2.1.0
+    - MYSQL_IMAGE=mysql:5.7 MONGO_IMAGE=mongo:4.0 POSTGRES_IMAGE=perconalab/percona-distribution-postgresql:11 PG_PRELOADED_LIBS=pg_stat_statements,pg_stat_monitor PMM_SERVER_IMAGE=percona/pmm-server:2.0.1
+    - MYSQL_IMAGE=mysql:8.0 MONGO_IMAGE=mongo:4.2 POSTGRES_IMAGE=perconalab/percona-distribution-postgresql:12 PG_PRELOADED_LIBS=pg_stat_statements,pg_stat_monitor PMM_SERVER_IMAGE=percona/pmm-server:2.1.0
```

Comment:
Is it "latest upstreams" as the comment above says, or that's "Percona" as comment below says?

Reply:
Yes, it was confusing. I added new group and left comment.

Reply:
Exactly. We did not have Percona PostgreSQL before, but we have it now, so it should be in the block below